src/ai/backend/storage/README.md

```diff
@@ -160,3 +162,12 @@ Please DO NOT remove Backend.AI mapped qtree in NetApp ONTAP Sysmgr GUI. If not,
 
 > NOTE:   
 Qtree name in configuration file(`storage-proxy.toml`) must have the same name created in NetApp ONTAP Sysmgr.
+
+### Weka.IO
+
+#### Prerequisites
+
+* Weka.IO agent installed and running
+* Weka.IO filesystem mounted under `/vfroot`
```

Comment:
`/vfroot` is an arbitrary sample text rather than a strict prerequisite, though it is used as the default config in the relative form by `install-dev.sh`.
Please rephrase it to indicate the general configuration option.